src/main/java/ua/danit/final_project/controllers/CommentController.java

```diff
@@ -9,6 +9,8 @@
 import ua.danit.final_project.entities.Comment;
 import ua.danit.final_project.services.CommentService;
 
+import org.springframework.web.bind.annotation.ModelAttribute;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.springframework.web.bind.annotation.ModelAttribute'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=34&issues=AWUkQ-lwOTG07Bqb8Z0V&open=AWUkQ-lwOTG07Bqb8Z0V)

[ISSUE_KEY]: AWUkQ-lwOTG07Bqb8Z0V